scripts/linux/electron-installer-appimage.sh

```diff
@@ -83,25 +83,6 @@ then
   usage
 fi
 
-function download_executable() {
-  local url=$1
-  local output=$2
-  wget "$url" -O "$output"
-  chmod +x "$output"
-}
-
-APPIMAGES_TAG=6
-APPIMAGES_GITHUB_RELEASE_BASE_URL=https://github.com/probonopd/AppImageKit/releases/download/$APPIMAGES_TAG
-
-if [ "$ARGV_ARCHITECTURE" == "x64" ]; then
-  APPIMAGES_ARCHITECTURE="x86_64"
-elif [ "$ARGV_ARCHITECTURE" == "x86" ]; then
-  APPIMAGES_ARCHITECTURE="i686"
-else
-  echo "Invalid architecture: $ARGV_ARCHITECTURE" 1>&2
-  exit 1
```

Comment:
Haha, I thought it didn't make sense anymore given that we had to create conditionals for the checksums. In any case, I came up with something even better. I'm updating the PR in a second...